### DISCUSSION TITLE
Opencomputers Compatibility Suggestions

### DISCUSSION THREAD

**Discussion by BallOfEnergy1** [HbmMods/Hbm-s-Nuclear-Tech-GIT#978] [General]
Anything you need/want in NTM relating to OC compatibility can be posted here and I'll add it to a list of things I'm adding to the mod. (no absurd/non-serious requests)

**Comment by diyamomd45** [comment]
RBMK Link

**Reply by BallOfEnergy1** [reply]
Can you be a bit more specific? Do you mean like a wireless RBMK link for OC? If so, that might be possible.

**Reply by diyamomd45** [reply]
Yes thats what I mean
